jenkins/promotion/promote-artifacts.jenkinsfile

```diff
@@ -54,7 +54,7 @@ pipeline {
             }
             steps {
                 script {
-                    currentBuild.description = "${DISTRIBUTION_BUILD_NUMBER} ${DISTRIBUTION_PLATFORM} ${DISTRIBUTION_ARCHITECTURE} ${INPUT_MANIFEST}"
+                    currentBuild.description = "${DISTRIBUTION_BUILD_NUMBER} ${DISTRIBUTION_PLATFORM} ${DISTRIBUTION_ARCHITECTURE} ${INPUT_MANIFEST} ${DISTRIBUTION_NAME}"
```

Comment:
Hi @Tokesh thanks for this tho it is also beneficial to add `${DISTRIBUTION_ARCHITECTURE}` as well.
Please update.

Thanks.

Reply:
> Hi @Tokesh thanks for this tho it is also beneficial to add `${DISTRIBUTION_ARCHITECTURE}` as well. Please update.
> 
> Thanks.

Hello @peterzhuamazon, but it already added. Can you recheck please?

Reply:
Oops, I must have missed that. Thanks.

Reply:
Can you add ${DISTRIBUTION_NAME}" before ${INPUT_MANIFEST} then?

Reply:
> Can you add ${DISTRIBUTION_NAME}" before ${INPUT_MANIFEST} then?

Of course! Its ready for your check

Reply:
Thanks, merged.